tokio/src/time/driver/mod.rs

```diff
@@ -118,28 +84,21 @@ use std::{cmp, fmt};
 /// [`turn`]: #method.turn
 /// [Handle.struct]: struct.Handle.html
 #[derive(Debug)]
-pub struct Timer<T> {
+pub(crate) struct Driver<T> {
```

Comment:
on the fence about this renaming...`Timer` feels more descriptive to me.

Reply:
We've been moving naming -> `[mod]::Driver` for a bit. We can re-evaluate in a follow up issue if you don't think it is a good direction.